package.json

```diff
@@ -7,30 +7,40 @@
   "author": "backmeupplz <backmeupplz@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "distribute": "yarn build && node dist/server.js",
-    "start": "tsc-watch --skipLibCheck --onSuccess 'node dist/server.js'",
     "build": "tsc --skipLibCheck",
+    "distribute-server": "node dist/server.js",
+    "distribute-checker": "node dist/checker.js",
+    "distribute": "yarn build && concurrently 'yarn distribute-server' 'yarn distribute-checker'",
```

Comment:
Why don't you use `yarn build && yarn distribute-server || yarn distribute-checker`?

Reply:
Because we need to run them concurrently